From 3b06d1045cc990a8610bdbd5c9b83440b1b6e932 Mon Sep 17 00:00:00 2001
From: Tomislav Vujec <tvujec@gmail.com>
Date: Tue, 17 Nov 2009 19:42:38 -0500
Subject: [PATCH] Don't warn about a missing session file

When started without session related command line parameters (e.g. from
gnome-session), metacity picks up client_id from the DESKTOP_AUTOSTART_ID
environment variable. Unfortunately, there is no way to distinguish if this was
passed from a config file, representing old saved session, or generated by
gnome-session, therefore load_state is attempted in each case. If the client_id
is generated, there will be no session file, and metacity will report a
warning.

Just remove the warning so that users won't always find a warning at the
start of their .xsession-errors.

https://bugzilla.gnome.org/show_bug.cgi?id=577576
---
 src/core/session.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/src/core/session.c b/src/core/session.c
index 7e3b389..32fda97 100644
--- a/src/core/session.c
+++ b/src/core/session.c
@@ -1098,8 +1098,6 @@ load_state (const char *previous_save_file)
         {
           /* oh, just give up */
 
-          meta_warning (_("Failed to read saved session file %s: %s\n"),
-                    canonical_session_file, error->message);
           g_error_free (error);
           g_free (session_file);
           g_free (canonical_session_file);
-- 
1.6.5.2